ui/pages/Search/components/VariantSearchForm.jsx

```diff
@@ -1,46 +1,87 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import { connect } from 'react-redux'
+import { Segment } from 'semantic-ui-react'
```

Comment:
This file now incorporates the code that was previously in the shared `VariantSearchFormContainer`